code/nrf-connect/samples/ble/prj.conf

```diff
@@ -20,8 +20,14 @@ CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y
 CONFIG_ASSERT=y
 
 # Application config - see all options in Kconfig.
-# CONFIG_PRST_BLE_ENCODING_BTHOME_V2=y
-# CONFIG_PRST_SLEEP_DURATION_MSEC=1000
+
+# The amount of time it sleeps between sending data.
+CONFIG_PRST_SLEEP_DURATION_MSEC=600000
+
+# Choose one of the following encoding options:
```

Comment:
I would refrain from exhaustively listing options in the `prj.conf` file. Reasoning:
- Why list some configs and not others?
- Why only in `prj.conf` and not `prj_debug.conf`?
- More importantly, this list will go stale once we add or remove some of these options.

A quick example and pointing to Kconfig for details is enough, as before.